lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -0,0 +1,317 @@
+/* global gettext */
+/* eslint one-var: ["error", "always"] */
+/* eslint no-alert: "error" */
+
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import FileUpload from './file_upload';
+import ShowErrors from './errors_list';
+import ShowProgress from './upload_progress';
+import LoggedInUser from './login_user';
+import LoggedOutUser from './logged_out_user';
+
+// TODO
+// edx zendesk APIs
+// access token
+// custom fields ids
+// https://openedx.atlassian.net/browse/LEARNER-2736
+// https://openedx.atlassian.net/browse/LEARNER-2735
+
+class RenderForm extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      fileList: [],
+      fileInProgress: null,
+      currentRequest: null,
+      errorList: [],
+    };
+    this.uploadFile = this.uploadFile.bind(this);
+    this.submitForm = this.submitForm.bind(this);
+  }
+
+  uploadFile(e) {
+    const url = 'https://arbisoft.zendesk.com/api/v2/uploads.json?filename=',
+      fileReader = new FileReader(),
+      request = new XMLHttpRequest(),
+      errorList = [],
+      $this = this,
+      file = e.target.files[0],
+      accessToken = 'd6ed06821334b6584dd9607d04007c281007324ed07e087879c9c44835c684da',
+      maxFileSize = 5000000,  // 5mb is max limit
```

Comment:
There's a linting error here. I believe you don't need two spaces between the comma and the start of the comment. I would reduce it to just one.